### DISCUSSION TITLE
2005/03/10/%E5%8E%BB%E5%B9%B4%E5%9C%A8%E5%AE%B6%E7%BB%83%E6%BA%9C%E5%86%B0%E7%9A%84%E7%85%A7%E7%89%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#7] [Announcements]
# 2005/03/10/%E5%8E%BB%E5%B9%B4%E5%9C%A8%E5%AE%B6%E7%BB%83%E6%BA%9C%E5%86%B0%E7%9A%84%E7%85%A7%E7%89%87

刚开始水平很差，两星期后就滑得很好了。

https://blog.qizhen.xyz/2005/03/10/%E5%8E%BB%E5%B9%B4%E5%9C%A8%E5%AE%B6%E7%BB%83%E6%BA%9C%E5%86%B0%E7%9A%84%E7%85%A7%E7%89%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
Unknown
2005年03月19日 21:37
你家住那里啊?


hp
2005年03月15日 09:29 
哈哈，用屁股也能滑啊？在虹口没见你用啊？不然租鞋的十块钱可以省了 :))

```
